react/src/js/react/posts/Post.jsx

```diff
@@ -97,6 +97,12 @@ class Post extends Reflux.Component {
                 contentList.push(<span key={`text-${index}`} className="post-text">{content}</span>);
             }
         });
+        if (this.props.images) {
```

Comment:
Wouldn't this cause images to be rendered twice on the post details page?

Reply:
fixed